camelot/view/controls/delegates/texteditdelegate.py

```diff
@@ -44,7 +43,7 @@ class TextEditDelegate(CustomDelegate, metaclass=DocumentationMetaclass):
 
     @classmethod
     def get_editor_class(cls):
-        return editors.TextEditEditor
+        raise NotImplementedError
```

Comment:
Is er een reden waarom hier geen None wordt teruggegeven? Deze editor gaan we niet meer gebruiken?

Reply:
die editor w denk ik nergens gebruikt, en die is denk ik ook niet correct, vandaar zou ik hem niet overzetten, maar moest het dus toch nog ergens bovenkomen ...